### PR TITLE
Add some libraries to the README.md

### PR DESCRIPTION
**Pull request by stephentoub**

### CODE REVIEW

**Comment by Maxwe11** [comment]
System.IO.FileSystem.Primitives lost :smile:

**Comment by stephentoub** [comment]
@Maxwe11, yes, there are still a few missing, including some of the XML ones.  I didn't say it was complete ;)

File: README.md [hunk 1]
```diff
@@ -99,12 +99,19 @@ The repo contains the following components. More libraries are coming soon. ['Wa
...
+
+* **System.Linq.Queryable**. Provides LINQ standard query operators that operate on objects that implement ```IQueryable<T>```.
+
 * **System.Linq.Parallel**.  Provides a parallelized implementation of LINQ to Objects. "Parallel LINQ" (PLINQ) 
   implements the full set of LINQ standard query operators as well as additional operators specific to parallel operations.
 
 * **System.Numerics.Vectors**. Provides a set of basic vector types that leverage single instruction, 
   multiple data (SIMD) CPU instructions. See our [recent][simd-post-1] [announcements][simd-post-2] for more details.
 
+* **System.ObjectModel**. Provides classes that can be used in the object model of a reusable library.
```
Comment (akoeplinger) [inline README.md]: That description doesn't really give me a clue what this library is about :smile:

Comment (stephentoub) [inline README.md]: Ok, what would? :)

Comment (akoeplinger) [inline README.md]: Geez, if I'd know what this does I wouldn't ask :laughing: 

The description in the [nuget package](https://www.nuget.org/packages/System.ObjectModel/4.0.10-beta-22605) seems better: `Provides types and interfaces that allow the creation of observable types that provide notifications to clients when changes are made to it.`

Comment (stephentoub) [inline README.md]: Thanks, @akoeplinger.  I used the NuGet package description, and added more libraries to the readme using the NuGet package description, or at least using them as the basis.

**Comment by stephentoub** [comment]
Feedback addressed.

**Comment by akoeplinger** [comment]
LGTM :+1:

**Comment by stephentoub** [comment]
@terrajobst or @richlander, look ok?

**Comment by ellismg** [comment]
LGTM.

**Pull request merged**